README.md

```diff
@@ -153,6 +153,12 @@ The easiest and preferred way to get going with SurrealDB on Unix operating syst
 curl -sSf https://install.surrealdb.com | sh
 ```
 
+If you want a binary newer than what's currently released, you can install the nightly one.
+
+```bash
+curl -sSf https://install.surrealdb.com | sh -s -- --nightly
```

Comment:
Should we use the same command as detailed in the README here: https://github.com/surrealdb/install.surrealdb.com ?

Reply:
Sure! In that case I will update the non-nightly one as well.

Reply:
Done